app/src/main/java/ci/sample/toandoan/circlecisample/MainActivity.java

```diff
@@ -4,10 +4,12 @@
 import android.os.Bundle;
 
 public class MainActivity extends AppCompatActivity {
+    private final static int a =10;
```

Comment:
ERROR: Static final field names must either be all caps (e.g. int HEIGHT_PX) for true constants, or start with s (e.g. AtomicInteger sNextId or Runnable sSuspendTask) for fields with mutable state or that dont feel like constants.